dnf-behave-tests/dnf/plugins-core/config-manager-addrepo.feature

```diff
@@ -0,0 +1,437 @@
+@dnf5
+Feature: dnf "config-manager" command - test "addrepo" subcommand
+
+
+Background:
+  Given I enable plugin "config_manager"
+    And I create file "/etc/yum.repos.d/repo1.repo" with
+        """
+        [repo1]
+        name=repo1 test repository
+        enabled=1
+        baseurl=http://something1.com/os/
+        """
+    And I create file "/etc/yum.repos.d/repo2.repo" with
```

Comment:
May be I overlooked something, but it looks like that repository is not used in any test in feature file. Is there any reason to create it?

Reply:
Repo2 was unused in this file. I deleted its creation.